frontend/app/src/i18n/cn.json

```diff
@@ -1285,26 +1286,5 @@
     "yourAccountIsSuspended": "您的账户暂时无法使用",
     "yourChats": "您的聊天",
     "yourRoleChanged": "{changedBy} 将您的角色从 {oldRole} 更改为 {newRole}",
-    "youreBlocked": "抱歉 - 您已被该群聊加入黑名单",
-    "videoCall": {
```

Comment:
Are these purposely removed?

Reply:
Looks wrong to me that those fields have been deleted.